addons/mail/static/src/components/thread_view/thread_view.js

```diff
@@ -85,6 +85,17 @@ class ThreadView extends Component {
         return this._messageListRef.comp.getScrollTop();
     }
 
+    /**
+     * @private
+     * @param {MouseEvent} ev
+     */
+    onScroll(ev) {
+        if (!this._messageListRef || !this._messageListRef.comp) {
```

Comment:
From how we use our other refs, apparently only `.comp` can be `undefined`, the ref object itself will always be set.